src/planner/index_scan_plan.cpp

```diff
@@ -55,8 +55,8 @@ IndexScanPlan::IndexScanPlan(storage::DataTable *table,
   //
   // Values that are left blank will be recorded for future binding
   // and their offset inside the value array will be remembered
-  index_predicate_.AddConjunctionScanPredicate(index_.get(), values_,
-                                               key_column_ids_, expr_types_);
+  //index_predicate_.AddConjunctionScanPredicate(index_.get(), values_,
+  //                                             key_column_ids_, expr_types_);
```

Comment:
Why don't we delete these lines?

Reply:
Goot catch! I forgot to do delete them. I'll do it.

Reply:
If we are shifting this from the planner to the execution engine, we won't be able to see the details of these predicates at the optimizer side, right? The plan returned by the optimizer to 'What-if index' will not have the required index plan details?